# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tcpflow/hotfix-inttypes.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

netviz/plot_view.h:134:42: error: 'uint64_t' has not been declared

--- tcpflow-1.6.1/src/netviz/plot_view.h.vanilla	2025-06-30 00:30:18.310999552 +0200
+++ tcpflow-1.6.1/src/netviz/plot_view.h	2025-06-30 00:30:25.087482050 +0200
@@ -26,6 +26,7 @@
 #include <vector>
 #include <string>
 #include <math.h>
+#include <inttypes.h>
 
 #ifndef M_PI
 #define M_PI		3.14159265358979323846
